src/test/java/org/homeschoolpebt/app/journeys/ApplyForSelfJourneyTest.java

```diff
@@ -69,13 +69,13 @@ void selfApplyFullFlow() {
     // Students subflow
     testPage.clickButton("Get started"); // "Student information" signpost
     assertPageTitle("Add a student");
-    testPage.enter("studentFirstName", "Stud");
-    testPage.enter("studentLastName", "McStudenty");
+    testPage.enter("studentFirstName", "Testy");
```

Comment:
Personally I preferred the old name ("Stud") since it made it clear it's a student. Not a blocker.